app/javascript/lib/auction.test.js

```diff
@@ -1,9 +1,22 @@
 import Action from './action';
+
+import Logger from '../src/Logger';
+
 import Auction from './auction';
 import Imperial from './imperial';
 import { Bond, Nation } from './constants';
 import auctionSetup from './auctionSetup';
 
+import BoardEurope from './board';
+
+const ImperialFromLog = (log) => {
+  const game = new Imperial(BoardEurope, new Logger('unittests', 'stub-test-id'));
```

Comment:
No, you're right, it doesn't need a logger

I guess I wanted to avoid having to null-check every time we call `this.#logger` inside `Imperial.js`, and having a stub logger was the best solution for that.

Writing wrappers methods within `Imperial.js` to perform the null-check seems a bit redundant, since that'd imply we'd need a new wrapper method for every method defined in `Logger`?

Reply:
Yeah that could work, too. Check my other comment about making a stub Logger as the default, which would not actually send logs through Logtail. If that works, it might be preferable.

Reply:
[76f61f0](https://github.com/Thrillberg/imperial/pull/729/commits/76f61f00f58e876e15be9e5e2d7e0ddfdbd8a432)